src/style/blocks/grade.css

```
@@ -0,0 +1,34 @@
+/* #region .grade */
+
+.grade {
+  display: flex;
+  gap: 19px;
+  margin-bottom: 24px;
+}
+
+.grade__stars4 {
+  display: flex;
+  margin-left: var(--card-between-size);
+}
+
+.grade__stars4-star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

```suggestion
  background: url(images/star-active.svg) no-repeat center;
```
https://www.w3schools.com/css/css_background_shorthand.asp